pkgs/tools/system/testdisk/default.nix

```diff
@@ -26,6 +26,11 @@ assert enableQt -> qwt != null;
     sha256 = "1zlh44w67py416hkvw6nrfmjickc2d43v51vcli5p374d5sw84ql";
   };
 
+  postPatch = ''
+    substituteInPlace linux/qphotorec.desktop \
+      --replace "/usr" "$out"
```

Comment:
desktop files can be copied out of the Nix store by desktop environments and thus become stale. The solution is to use relative paths, not abspaths.

Reply:
Will fix.

What DE does that? Do you have a link to some doc describing this behavior?

Reply:
Autostart specification: https://specifications.freedesktop.org/autostart-spec/autostart-spec-latest.html.